### PR TITLE
Skyline/work/20191118 group comparisons rebased

### PR DESCRIPTION
**Pull request by chambm**
Rebased version of @pvvemul 's branch. I'm not sure it even compiles because I wasn't familiar with the code I was doing conflict resolution on. I left out the last merge conflict commit because it didn't seem necessary after rebase and every file was conflicted.

### CODE REVIEW

**Comment by chambm** [comment]
I forgot draft PRs don't get built on TeamCity.

**Comment by chambm** [comment]
@pvvemul Are you able to run the failing perf/tutorial tests locally?

**Comment by pvvemul** [comment]
@chambm I fixed a bug causing some of the tutorial tests to fail. TestAreaCVQValuesAndRatios is passing on my machine though.

**Comment by pvvemul** [comment]
@chambm The merge that you made seemed to delete one of the <data> tags so Resources.resx is invalid, how do I fix it?

**Comment by pvvemul** [comment]
@chambm nevermind, I fixed it and committed.

File: pwiz_tools/Skyline/TestFunctional/AreaCVHistogramTest.cs [hunk 1]
```diff
@@ -377,10 +377,13 @@ private void TestRefinement()
             var refineDlg = ShowDialog<RefineDlg>(() => SkylineWindow.ShowRefineDlg());
             RunUI(() => { refineDlg.CVCutoff = 20; });
             OkDialog(refineDlg, refineDlg.OkDialog);
-            var doc = SkylineWindow.Document;
-            var refineDocState = (doc.PeptideGroupCount, doc.PeptideCount, doc.PeptideTransitionGroupCount,
-                doc.PeptideTransitionCount);
-            Assert.AreEqual(graphStates[0], refineDocState);
+            WaitForCondition(() =>
```
Comment (nickshulman) [inline pwiz_tools/Skyline/TestFunctional/AreaCVHistogramTest.cs]: If you wanted to, a different way to check whether SkylineWindow.ShowRefineDlg has finished its work is to do a "WaitForDocumentChange".

But, this way works too.

**Pull request merged**